sscanApp/src/saveData_writeXDR.c

```diff
@@ -3565,13 +3567,19 @@ LOCAL void remount_file_system(char* filesystem)
 		}
 		*cout='\0';
 		
+		/* Enable write-through cache */
+		nfs3CacheOptions = 1;
+		
 		/* Mount the new file system */
 		if (nfsMount(hostname, filesystem, "/data")==ERROR) {
 			strncpy(msg, "Unable to mount file system !!!!", MAX_STRING_SIZE);
 		} else {
 			file_system_state= FS_MOUNTED;
 			path = local_pathname;
 		}
+		
+		/* Disable write-through cache */
+		nfs3CacheOptions = 0;
```

Comment:
It would be better to save the previous value of `nfs3CacheOptions` before overwriting it, then restore that value after the mount. Also instead of '1' you could `#include "nfsCommon.h"` and use the macro `NFS_CACHE_WRITE_THROUGH`.

Reply:
Sorry, that should be `#include "nfs/nfsCommon.h"`